micrometer/pom.xml

```diff
@@ -0,0 +1,312 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.wildfly.quickstarts</groupId>
+        <artifactId>wildfly-quickstart-parent</artifactId>
+        <!--
+        Maintain separation between the artifact id and the version to help prevent
+        merge conflicts between commits changing the GA and those changing the V.
+        -->
+        <version>5</version>
+        <relativePath/>
+    </parent>
+
+    <artifactId>micrometer</artifactId>
+    <version>30.0.0.Final-SNAPSHOT</version>
+    <packaging>war</packaging>
+
+    <name>Quickstart: micrometer</name>
+
+    <properties>
+        <version.server>30.0.0.Final-SNAPSHOT</version.server>
```

Comment:
Please add the comments describing and splitting the version groups
        <!-- the version for the Server -->
        <version.server>30.0.0.Beta1</version.server>
        <!-- the versions for BOMs, Packs and Plugins -->
        <version.bom.ee>${version.server}</version.bom.ee>
        <version.plugin.wildfly>4.2.0.Final</version.plugin.wildfly>
        <version.plugin.wildfly-jar>10.0.0.Final</version.plugin.wildfly-jar>
        <!-- The versions for other dependencies and plugins -->
        <version.testcontainers>1.18.3</version.testcontainers>

Reply:
I also don't follow here. This seems to match what you did here: https://github.com/emmartins/quickstart/blob/2a2750ae8e78c42c2b1dbba92f6baca12a67ea4d/microprofile-openapi/pom.xml

Reply:
I just mean the comments should be added too, e.g. `<!-- the version for the Server -->`